ui/packages/atlasmap/src/UI/ColumnMapper/Column.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-import { boolean, number } from "@storybook/addon-knobs";
+import { boolean, number, text } from "@storybook/addon-knobs";
```

Comment:
`text` is unused and makes CI fail

Reply:
Oh, missed that one, I will check the CI results next time. Should be OK now.